### PR TITLE
[FR] Update `Help centre/Account restrictions`

### PR DESCRIPTION
**Pull request by Pisapou**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Help_centre/Account_restrictions/fr.md [hunk 1]
```diff
@@ -73,7 +70,7 @@ Si nous décidons de vous laisser revenir, nous vous demanderons que vous accept
 - **Vous ne vous associerez pas avec d'autres comptes.** Cela inclut de jouer sur un ordinateur partagé.
 - **Le moindre mauvais comportement majeur aura pour résultat une restriction immédiate.** Cela inclut les silences majeurs.
 
-Dans certains cas, un utilisateur qui revient d'une restriction peut être banni des tournois officiels, à la fois en tant que joueur et membre de l'équipe du tournoi. La durée de ce bannissement est d'un an par restriction (si l'utilisateur en est à sa deuxième restriction, le bannissement durera 2 ans). L'équipe de support aux comptes clarifiera la durée de ce bannissement.
+Les retours d'appel dans les conditions standard ci-dessus sont également interdits de tournois officiels, à la fois en tant que joueur et membre de l'équipe du tournoi. La durée de ce bannissement est d'un an par restriction (si l'utilisateur en est à sa deuxième restriction, le bannissement durera 2 ans). L'équipe de support aux comptes clarifiera la durée de ce bannissement.
```
Comment (Biokax) [inline wiki/Help_centre/Account_restrictions/fr.md:73]:
```suggestion
Les retours d'appel dans les conditions standard ci-dessus sont également interdits de tournois officiels et badgés, à la fois en tant que joueur et membre de l'équipe du tournoi. La durée de ce bannissement est d'un an par restriction (si l'utilisateur en est à sa deuxième restriction, le bannissement durera 2 ans). L'équipe de support aux comptes clarifiera la durée de ce bannissement.
```

Comment (Biokax) [reply wiki/Help_centre/Account_restrictions/fr.md:73]: il manquait un bout

**Review by Biokax — APPROVED** [review]

File: wiki/Help_centre/Account_restrictions/fr.md [hunk 2]
```diff
@@ -73,7 +70,7 @@ Si nous décidons de vous laisser revenir, nous vous demanderons que vous accept
 - **Vous ne vous associerez pas avec d'autres comptes.** Cela inclut de jouer sur un ordinateur partagé.
 - **Le moindre mauvais comportement majeur aura pour résultat une restriction immédiate.** Cela inclut les silences majeurs.
 
-Dans certains cas, un utilisateur qui revient d'une restriction peut être banni des tournois officiels, à la fois en tant que joueur et membre de l'équipe du tournoi. La durée de ce bannissement est d'un an par restriction (si l'utilisateur en est à sa deuxième restriction, le bannissement durera 2 ans). L'équipe de support aux comptes clarifiera la durée de ce bannissement.
+Les retours d'appel dans les conditions standard ci-dessus sont également interdits de tournois officiels et badgés, à la fois en tant que joueur et membre de l'équipe du tournoi. La durée de ce bannissement est d'un an par restriction (si l'utilisateur en est à sa deuxième restriction, le bannissement durera 2 ans). L'équipe de support aux comptes clarifiera la durée de ce bannissement.
```
Comment (Ganondorf-1) [inline wiki/Help_centre/Account_restrictions/fr.md:73]:
```suggestion
Les utilisateurs qui sortent d'une restriction dans les conditions standard ci-dessus sont également interdits de participer aux tournois officiels et badgés, à la fois en tant que joueur et membre de l'équipe du tournoi. La durée de ce bannissement est d'un an par restriction (si l'utilisateur en est à sa deuxième restriction, le bannissement durera 2 ans). L'équipe de support aux comptes clarifiera la durée de ce bannissement.
```

File: wiki/Help_centre/Account_restrictions/fr.md [hunk 3]
```diff
@@ -73,7 +70,7 @@ Si nous décidons de vous laisser revenir, nous vous demanderons que vous accept
...
 
 **Tricher ou partager son compte durant un tournoi officiel mènera à un bannissement permanent de l'utilisateur.** Vous ne pouvez pas faire appel quant à ce bannissement (votre mail sera ignoré si vous essayez) et ne sera pas supprimé, réduit ou levé dans n'importe quelles circonstances.
```
Comment (Ganondorf-1) [inline wiki/Help_centre/Account_restrictions/fr.md:75]:
```suggestion
**Tricher ou partager son compte durant un tournoi officiel mènera à un bannissement permanent de l'utilisateur.** Vous ne pouvez pas faire appel quant à ce bannissement (votre mail sera ignoré si vous essayez) et il ne sera en aucun cas supprimé, réduit ou levé.
```

File: wiki/Help_centre/Account_restrictions/fr.md [hunk 4]
```diff
@@ -13,6 +8,8 @@ Lorsqu'un compte est restreint, il ne peut pas intéragir avec la communauté ju
 
 Les fonctionnalités suivantes sont désactivées pour les comptes restreints :
 
+- La possibilité de participer à des concours officiels
+- La ppossibilité de participer à des salles multijoueurs
```
Comment (Ganondorf-1) [inline wiki/Help_centre/Account_restrictions/fr.md:12]:
```suggestion
- La possibilité de participer à des salles multijoueurs
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

**Review by Ganondorf-1 — APPROVED** [review]

**Pull request merged**